pkg/exporter/sumologicexporter/sender.go

```diff
@@ -98,19 +92,30 @@ func (b *bodyBuilder) Reset() {
 }
 
 // addLine adds line to builder and increments counter
-func (b *bodyBuilder) addLine(line string) error {
-	_, err := b.builder.WriteString(line)
-	if err != nil {
-		return err
-	}
+func (b *bodyBuilder) addLine(line string) {
+	b.builder.WriteString(line) // WriteString can't actually return an error
```

Comment:
https://go.dev/src/strings/builder.go
```
// WriteString appends the contents of s to b's buffer.

// It returns the length of s and a nil error.

func (b *Builder) WriteString(s string) (int, error) {

	b.copyCheck()

	b.buf = append(b.buf, s...)

	return len(s), nil

}
```

Reply:
Yes, that's what I mean, there's no point to checking the error.

Reply:
I just pointed it for reference :+1: